src/views/FacilityDetails.vue

```diff
@@ -354,7 +354,7 @@
                 </ion-card-title>
               </ion-card-header>
               <ion-item lines="full">
-                <ion-label>{{ translate('Identification') }}</ion-label>
+                <ion-label slot="start">{{ translate('Identification') }}</ion-label>
```

Comment:
why do we need to slot this label?

Reply:
Since we have used slot "end" in another label showing value of this identification. Ionic by default consider that as primary element of card and truncate this label. By adding slot start in the first label, it truncate the right label.

Reply:
If we remove the slot end from the second label, then that label will start from the center of the card.